### PR TITLE
Introduce presence changing

### PR DESCRIPTION
**Pull request by LWTechGaming**
Added command to change presence in Discord + added command into readme.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ Command | Usage | SB<sup>1</sup> | DB<sup>2</sup> | C<sup>3</sup>
 `tag <name>` | Triggers a tag || Yes |
 `cat` | Sends a random cat image |||
 `youtube <query>` | Searches YouTube and returns results |||
+`presence <online/idle/dnd/invisible>` | Changes your presence in Discord |||
```
Comment (jaydenkieran) [inline readme.md]: We should probably remove `your` here as this command can be ran while the bot isn't a selfbot.

File: turbo/commands.py [hunk 2]
```diff
@@ -460,3 +460,29 @@ def creator_only(func):
             response += "\n{0}`{1}` - <https://youtube.com{2}>".format(prefix, l['title'], l['href'])
             amount -= 1
         return Response(response)
+    
+    async def c_presence(self, option=None):
+        """
+        Changes your presence on Discord
+
+        {prefix}presence online/idle/dnd/invisible
```
Comment (jaydenkieran) [inline turbo/commands.py]: Surround the valid arguments with `<` and `>` 👍

File: turbo/commands.py [hunk 3]
```diff
@@ -460,3 +460,29 @@ def creator_only(func):
...
+        
+        Invisible makes you appear offline.
+        Optionally use do_not_disturb for dnd if you really want to make it long.
```
Comment (jaydenkieran) [inline turbo/commands.py]: Using `do_not_disturb` wouldn't actually work, it takes enum values only, `dnd` is an enum value, `do_not_disturb` is not. (https://github.com/Rapptz/discord.py/blob/master/discord/enums.py)

File: readme.md (hunk 1 above)
Comment (LWTechGaming) [inline readme.md]: Shrug, why not.

**Pull request merged**